src/server/ideController.py

```diff
@@ -26,40 +28,258 @@ def __init__(self, template_path, logger):
 
   @cherrypy.expose
   def index(self):
-    return "Hello world"
-    # TODO Return page/template render for the IDE part
-    # tmpl = loader.load('edit.html')
-    # # set args in generate as key1=val1, key2=val2
-    # stream = tmpl.generate()
-    # return stream.render('html')
+    """
+    IDEController index page generator
+    (Path : /ide/ -- /ide/index)
+
+    @return: Template HTML render
+    """
+    if not cherrypy.session.get('username'):
+      cherrypy.session['username'] = uuid.uuid4()  # XXX Session should be set by the id/auth module
```

Comment:
I had a POC in progress about this : [auth.py](https://github.com/grondinjc/wide_poc/blob/login/login/src/authentication/auth.py) and [usage](https://github.com/grondinjc/wide_poc/blob/login/login/src/index.py#L18)